src/verify/table_report_policy.sql

```diff
@@ -0,0 +1,25 @@
+-- Verify maevsi:table_report_policy on pg
+
+BEGIN;
+
+DO $$
+BEGIN
+  ASSERT (SELECT pg_catalog.has_table_privilege('maevsi_account', 'maevsi.report', 'INSERT'));
+  ASSERT (SELECT pg_catalog.has_table_privilege('maevsi_account', 'maevsi.report', 'SELECT'));
+  ASSERT NOT (SELECT pg_catalog.has_table_privilege('maevsi_account', 'maevsi.report', 'UPDATE'));
+  ASSERT NOT (SELECT pg_catalog.has_table_privilege('maevsi_account', 'maevsi.report', 'DELETE'));
```

Comment:
Shouldn't we allow reports to be updated or deleted by their authors?

Reply:
Monitoring page of all blocking requests is enough, unblock page nice-to-have but not legally needed, could be covered by support ticket.

Reply:
Reports can now be created and read by who created them. No updated or deletions possible. Just as sidenote: this is not about blocking, but about reports. PR for blocking is this: https://github.com/maevsi/sqitch/pull/73